docs/DafnyRef/Types.md

```diff
@@ -1274,7 +1274,7 @@ type.  For such assignments, an explicit conversion must be used, see
 
 Dafny builds in three families of subset types, as described next.
 
-### 11.3.1 Type`nat`
```

Comment:
This is a mistake right?

Reply:
Thanks, I assumed that this wasn't my fault. I'll revert this and update the manual pdf.